RealmSwift/List.swift

```diff
@@ -314,14 +314,16 @@ public final class List<T: Object>: ListBase {
 
      - parameter index: The index at which to remove the object.
      */
-    public func remove(objectAtIndex index: Int) {
+    public func remove(at index: Int) {
```

Comment:
The Swift standard library version of this method returns the removed element. I don't know if this is something we should replicate.